src/devices/Camera/Camera/CameraInfo.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Camera;
+
+/// <summary>
+/// Information on the camera listed by the tools
+/// </summary>
+public record class CameraInfo(
+    int Index,
+    string Name,
+    string MaxResolution,
+    string DevicePath)
+{
+    private const string Line1 = "Available cameras";
+    private const string Line2 = "-----------------";
+
+    /// <summary>
+    /// Parse the string obtained from the --list-cameras command line and
+    /// extracts the main characteristics
+    /// </summary>
+    public static async Task<IEnumerable<CameraInfo>> From(string listOutputString)
```

Comment:
you're missing the intellisense comments for the parameters here and the return type